packages/external-db-firestore/lib/sql_filter_transformer.js

```diff
@@ -130,6 +134,19 @@ class FilterParser {
         return fieldName
     }
 
+    parseStringBegins(fieldName, inlineFields, value) {
+        return [{
+            fieldName: this.inlineVariableIfNeeded(fieldName, inlineFields),
+            opStr: '>=',
+            value,
+        },
+        {
+            fieldName: this.inlineVariableIfNeeded(fieldName, inlineFields),
+            opStr: '<',
+            value: value + String.fromCharCode(65535)
```

Comment:
`String.fromCharCode(65535)` should be at firestore_utils as `MaxChar` in my opinion

Reply:
pack this value to a function or const with proper name so we can understand what you did there.